widgets/nuxeo-document-suggestion.html

```diff
@@ -272,6 +272,9 @@
         }
 
         _selectionFormatter(doc) {
+          if (!doc['entity-type'] && typeof doc === 'string') {
```

Comment:
if doc is a string, then `!doc['entity-type']` will always be `true`:
```suggestion
          if (typeof doc === 'string') {
```